distribution/lib/Standard/Base/0.0.0-dev/src/Data/Java_Json.enso

```diff
@@ -1,226 +0,0 @@
-    at : Text -> Jackson_Object | Boolean | Number | Nothing | Text | Vector ! No_Such_Key
-    at self key:Text = self.get key (Error.throw (No_Such_Key.Error self key))
-
-    ## GROUP Metadata
-       ICON metadata
-       Get the keys of the object.
-    field_names : Vector
-    field_names self = self.field_array
-
-    ## ICON dataframe_map_column
-       Maps a function over each value in this object
-
-       Arguments:
-       - function: The function to apply to each value in the map, taking a
-         value and returning a value.
-    map : (Any->Any) -> Vector
-    map self function =
-        kv_func = _ -> function
-        self.map_with_key kv_func
-
-    ## Maps a function over each field-value pair in the object.
-
-       Arguments:
-       - function: The function to apply to each key and value in the map,
-         taking a key and a value and returning a value.
-    map_with_key : (Any -> Any -> Any) -> Vector
-    map_with_key self function =
-        self.field_names.map key->
-            value = self.get key
-            function key value
-
-    ## GROUP Conversions
-       ICON convert
-       Convert the object to a Vector of Pairs.
-    to_vector : Vector
-    to_vector self =
-        keys = self.field_array
-        proxy = Array_Proxy.new keys.length (i-> [(keys.at i), (self.get (keys.at i))])
-        Vector.from_polyglot_array proxy
-
-    ## GROUP Metadata
-       ICON metadata
-       Gets the number of keys in the object.
-    length : Number
-    length self = self.object_node.size
-
-    ## GROUP Logical
-       ICON metadata
-       Returns True iff the Map is empty, i.e., does not have any entries.
-    is_empty : Boolean
-    is_empty self = self.length == 0
-
-    ## GROUP Logical
-       ICON metadata
-       Returns True iff the Map is not empty, i.e., has at least one entry.
-    not_empty : Boolean
-    not_empty self = self.is_empty.not
-
-    ## PRIVATE
-       Convert the object to a JS_Object.
-    to_js_object : JS_Object
-    to_js_object self =
-        pairs = self.field_names.map name-> [name, self.at name . to_js_object]
-        JS_Object.from_pairs pairs
-
-    ## PRIVATE
-       Convert to a Text.
-    to_text : Text
-    to_text self = self.to_json
-
-    ## PRIVATE
-       Convert JS_Object to a friendly string.
-    to_display_text : Text
-    to_display_text self =
-        self.to_text.to_display_text
-
-    ## PRIVATE
-       Convert to a JSON representation.
-    to_json : Text
-    to_json self = self.object_node.toString
-
-## PRIVATE
-   Make a field name selector
-make_field_name_selector : Jackson_Object -> Display -> Widget
-make_field_name_selector js_object display=Display.Always =
-    Single_Choice display=display values=(js_object.field_names.map n->(Option n n.pretty))
-
-## ICON convert
-   Extension for Text to allow use.
-Text.parse_fast_json : Nothing | Boolean | Number | Text | Vector | Jackson_Object
-Text.parse_fast_json self = Java_Json.parse self
```

Comment:
Can we add this to Extensions and make it call `Text.parse_json` with a deprecated warning attached.